react/components/SKUSelector/components/SelectorItem.tsx

```diff
@@ -110,7 +113,9 @@ function SelectorItem({
       tabIndex={0}
       onClick={onClick}
       style={containerStyles}
-      className={containerClasses}
+      className={`${containerClasses} ${
+        !isAvailable ? handles.skuItemNotAvailable : handles.skuItemAvailable
+      }`}
```

Comment:
I think it would be more clear if this were a modifier on `containerClasses`, just like `slug` or `isSelected` inside the `useMemo` hook.